test/voracle/TestLag.sol

```diff
@@ -0,0 +1,193 @@
+pragma solidity ^0.5.0;
+
+import "../../contracts/voracle/Lag.sol";
+import "../mocks/MockIPRS.sol";
+import "truffle/Assert.sol";
+
+contract TestLag {
+    Lag public lag;
+    Lag public lagNoneGov;
+    MockIPRS public mockIPRS;
+    MockIPRS public mockIPRS2;
+
+    function beforeEach() public {
+        mockIPRS = new MockIPRS(10);
+        mockIPRS2 = new MockIPRS(11);
+        lag = new Lag(address(this), address(0));
+        lagNoneGov = new Lag(address(1), address(2));
+    }
+
+    function testHalt() public {
+        lag.halt();
+
+        Assert.isTrue(lag.halted(), "lag.halted are should be true");
+    }
+
+    function testHaltWhenNotGov() public {
+        (bool r,) = address(lagNoneGov).call(abi.encodePacked(lagNoneGov.halt.selector));
+
+        Assert.equal(r, false, "lag.halted are should be false");
+    }
+
+    function testResume() public {
+        lag.resume();
```

Comment:
should assert lag.halted first to double check that halt is false and then use lag.halt() then assert if halt is true then do lag.resume and assert lag.halt again